tests/security/cc/ipsec/ipsec_client.pm

```diff
@@ -42,6 +42,7 @@ sub run {
     # Test the IPSec connection
     assert_script_run('ipsec start');
     assert_script_run('stime=$(date +\'%H:%M:%S\')');
+    sleep 60;
```

Comment:
is there a way to not rely on `sleep`, but rather check something either in the logs or poll if a device is created? I can imagine that on heavy-loaded hosts, this could fail...

Reply:
> is there a way to not rely on `sleep`, but rather check something either in the logs or poll if a device is created? I can imagine that on heavy-loaded hosts, this could fail...

Hi Santi, we are doing it really temporarily because it fails all week in maintenance. Next week we will push another workaround, which avoids sleep :)

Reply:
make it a soft fail with a reference to the ticket and I'm good :D

Reply:
> record_info("The addition of the sleep command is a temporary workaround.", "It will be replaced soon :)");

still a reference to a  ticket is missing... (a soft-failure would be better)

Reply:
done!